### PR TITLE
feat(upload-media): Upload Media View Component

### PR DESCRIPTION
**Pull request by jwicks31**

### CODE REVIEW

**Comment by ericelliott** [comment]
Where's the demo?

**Comment by jwicks31** [comment]
Missing Blue Header Bar

**Comment by jwicks31** [comment]
Justify Upload Media Title to Left

**Comment by jwicks31** [comment]
Ensure thumbnails crop only, no distortion of images

**Comment by jwicks31** [comment]
Description Text is too bold

**Comment by jwicks31** [comment]
Add margin after Save & Continue Button

**Comment by jwicks31** [comment]
Lets do our own custom meter and get rid of the grommet package.

**Comment by ericelliott** [comment]
* [ ] Ditch grommet - make our own progress meter
* [ ] Ditch styled components
* [ ] The custom file thumbnail preview needs to be an SVG you can color, and the user should be able to replace the entire SVG
* [ ] All styles should be modifiable by the user
* [ ] User should be able to provide their own custom view and use the functions you provide

File: public/images/file-uploaded.png
Comment (jwicks31) [inline public/images/file-uploaded.png:1]: Lets get an SVG that we can style the colors on and allow it to be replaced.

**Comment by jwicks31** [comment]
All styles should be modifiable - meaning colors of all buttons, text. Sizes + padding of all buttons and text. (any arbitrary css)

**Pull request closed without merging**